docs/custom-commands.md

```diff
@@ -6,15 +6,15 @@ It is possible to extend fin with custom commands (per project).
 
     File `.docksal/commands/updb` (Notice **no extension**. Script name should match command name)
 
-    ```bash
+```bash
```

Comment:
This is what happens. Attached screenshot.

![screen shot 2016-11-23 at 11 38 42 am](https://cloud.githubusercontent.com/assets/1504038/20570409/7e9e1ae0-b171-11e6-8c59-19251d212652.png)